supplementary_style_guide/style_guidelines/code-commands.adoc

```diff
@@ -39,6 +39,25 @@ Some tasks in this procedure require root privileges, which you can get temporar
 .Additional resources
 * link:https://www.redhat.com/sysadmin/difference-between-sudo-su[Exploring the differences between sudo and su commands in Linux]
 
+[[ellipses-in-yaml-code-blocks]]
+== Ellipses in YAML code blocks
+
+Use the pound sign (`#`) to comment out an ellipsis in YAML code blocks.
+YAML reserves `...` to indicate the end of a document without starting a new document.
+
+.Example AsciiDoc
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1alpha1
+kind: CertManager
+metadata:
+  name: cluster
+# ...
+----
+
```

Comment:
It would be helpful to add some text to show how that code is rendered, similar to the example in the [Commands requiring root privileges](https://redhat-documentation.github.io/supplementary-style-guide/#commands-with-root-privileges) section.

Reply:
My preview shows the code rendering as a YAML code block.